.github/workflows/package-and-deploy.yml

```diff
@@ -0,0 +1,112 @@
+name: Package and deploy everest-admin-panel
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      tag:
```

Comment:
Because the release step now only runs when there is a tag, the input here does not really make sense anymore.